src/ui/components/inputs/EmailInputWithSpellingHelp/SpellingHelp.tsx

```diff
@@ -29,11 +29,17 @@ export const SpellingHelp = ({ suggestedEmail, onEmailChange, onSpellingHelpPres
 
   if (!showBanner) return null
 
+  const emailMessage =
+    suggestedEmail?.address && suggestedEmail?.domain
+      ? `Veux-tu plutôt dire ${suggestedEmail.address}@${suggestedEmail.domain}\u00a0?`
+      : ''
+
+  if (emailMessage === '') return null
```

Comment:
je propose de faire l'early return sur les variables qui nous intéressent vraiment plutot qu'une conséquence de ces variables

```suggestion
  if (!suggestedEmail?.address || !suggestedEmail?.domain) return null

  const emailMessage = `Veux-tu plutôt dire ${suggestedEmail.address}@${suggestedEmail.domain}\u00a0?`
```

et ça m'a fait aller voir le type de la variable `suggestedEmail`, c'est `suggestedEmail?: MailSuggestion` et tout les champs de `MailSuggestion` sont obligatoires

donc on peut encore simplifier en

```suggestion
  if (!suggestedEmail) return null

  const emailMessage = `Veux-tu plutôt dire ${suggestedEmail.address}@${suggestedEmail.domain}\u00a0?`
```